source/index.html.md

```diff
@@ -540,26 +527,103 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
+    <span class="type">(<code>storing</code>|<code>using</code>)</span>
+  </dt>
+  <dd>
+    Indicate if the payment credential (e.g. PAN and expiry) will be stored for
+    a later transaction where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>storing</code>: The payment credential will be stored; may only be
+    stored if the authorization is approved.
+    <br />
+    <code>using</code>: The payment credential was already stored and are now
+    being used.
+    <br />
+    If <code>initiator</code> is <code>merchant</code> then
+    <code>storing</code> is invalid and <code>using</code> is assumed.
+    For a first-in-series authorization, <code>storing</code> is assumed.
+
+    <div class="type">Optional</code>
+  </dd>
+
+  <dt>initiator
+    <span class="type">(<code>cardholder</code>|<code>merchant</code>)</span>
+  </dt>
+  <dd>
+    The initiator of the authorization. An authorization is initiated by the
+    cardholder if the cardholder decided the transaction should be created.
+    This is regardless of whether stored payment credential is being used.
+    <br />
+    For compliance reasons there should be a previous approved transaction
+    marked with <code>storing</code> before <code>initiator</code> may be
+    <code>merchant</code>.
+    For subsequent-in-series authorizations, <code>merchant</code> is assumed;
```

Comment:
```suggestion
    For subsequent-in-series authorizations, if <code>initiator</code> is left out, <code>merchant</code> is assumed;
```

To be in line with my previous suggestions regarding lines 553-554. I don't really like the phrase "left out" :man_shrugging:

Reply:
Fixed by https://github.com/clearhaus/gateway-api-docs/pull/111/commits/e4a6aebaf5e53b32346e1cb0199460b395865f85#diff-d30d91dbcd59693b69f9242acb4630c5R601-R606